docs/src/pages/[platform]/connected-components/authenticator/customization/customization.i18n.react-native.mdx

```diff
@@ -1 +1,47 @@
-TODO: REACT NATIVE DOCS
```

Comment:
Ideally, we would be able to use the same fragments for more than one platform to prevent duplication and the deviation that will come after